configs/nvme-storage-configurator/base/configure.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Enable debugging
+set -eux
+
+if [ -f /host-etc-systemd-dir/nvme-init-done ]; then
+    echo 'NVMe init already done'
+    exit 0
+fi
+
+cp -f /scripts/configuration.sh /host-usr-local-bin/configuration.sh
+chmod 0755 /host-usr-local-bin/configuration.sh
+
+cp -f /scripts/nvme-storage.service /host-etc-systemd-dir/nvme-storage.service
+chmod 0644 /host-etc-systemd-dir/nvme-storage.service
+
+nsenter -t 1 -m -u -i -n -p -- systemctl daemon-reload
+nsenter -t 1 -m -u -i -n -p -- systemctl enable nvme-storage.service
+touch /host-etc-systemd-dir/nvme-init-done
+nsenter -t 1 -m -u -i -n -p -- systemctl reboot
```

Comment:
can we do this? don't we need to evacuate  the pods first from the node (and disable scheduling for it)?

Reply:
also, why do we need a reboot?

Reply:
This configuration requires reboot as kubelet service has dependency on `/var/lib/kubelet/pods` and `/var/lib/containers`. We can add cordoning and draining option for the node and uncordon them after the reboot.

Reply:
I have addeded the following commands to gracefully reboot the nodes --

```
oc adm cordon $NODE_NAME
oc adm drain $NODE_NAME --delete-emptydir-data --ignore-daemonsets --grace-period=0 --force
```

and after the node reboots 

`oc adm uncordon $NODE_NAME`